easybuild/easyconfigs/c/cairo/cairo-1.14.6-foss-2016a.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = "cairo"
+version = '1.14.6'
+
+homepage = 'http://cairographics.org'
+description = """Cairo is a 2D graphics library with support for multiple output devices.
+ Currently supported output targets include the X Window System (via both Xlib and XCB), Quartz, Win32, image buffers,
+ PostScript, PDF, and SVG file output. Experimental backends include OpenGL, BeOS, OS/2, and DirectFB"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['http://cairographics.org/releases/']
+sources = [SOURCE_TAR_XZ]
+
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('libpng', '1.6.21'),
+    ('freetype', '2.6.2'),
+    ('pixman', '0.34.0'),
+    ('fontconfig', '2.11.94'),
+    ('expat', '2.1.0'),
```

Comment:
@SimonPinches our most recent `cairo` easyconfigs also have `GLib` and `libX11` listed as dependencies; did you remove these intentionally here?

cfr. https://github.com/hpcugent/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/c/cairo/cairo-1.14.6-intel-2016a.eb